pkg/validation/test_test.go

```diff
@@ -684,7 +684,7 @@ func TestValidateTests(t *testing.T) {
 	} {
 		t.Run(tc.id, func(t *testing.T) {
 			v := newSingleUseValidator()
-			errs := v.validateTestStepConfiguration(NewConfigContext(), "tests", tc.tests, tc.release, tc.releases, sets.New[string](), tc.resolved)
+			errs := v.validateTestStepConfiguration(NewConfigContext(), "tests", tc.tests, tc.release, nil, tc.releases, sets.New[string](), tc.resolved)
```

Comment:
Don't leave commits that don't compile in the `git` history — unless you're going to use the `rebase` merge method, which I don't recommend.  They make `revert`, `bisect`, etc. more difficult to use.

c822c03d, 5f0c26de, and 38479a69 should be combined into one commit.